examples/mujoco/scenarioSimpleDocking.py

```diff
@@ -0,0 +1,175 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2025, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+"""
+It's recommended to review the following scenario(s) first (and any
+recommended scenario(s) that they may have):
+
+#. examples/mujoco/scenarioArmWithThrusters.py
+
+This scripts shows how one can simulate two independent spacecraft
+docking and constraining their movement with respect to each other.
+This illustrates the use of MuJoCo `equality` in `MJScene`.
+
+The multi-body system is defined in the XML file `sats_dock.xml`.
+The system defined in this XML file is simple, but it illustrates some
+of the tools MuJoCo offers for our simulation needs. First, this shows
+how contact physics can be disabled through `flag contact="disable"`.
+Second, it illustrates how two free-floating bodies can be defined.
+Third, it shows how to declare an `equality`.
+
+Equalities are a way to add dynamical constraints to a system. In this
+case, we use a 'weld' constraint between two sites, which tells MuJoCo
+to enforce that the frames associated with the sites are aligned. This
+prevents any relative movement between them, and thus between the bodies
+that they are attached to. We use this to simulate the dynamics after
+docking. MuJoCo will automatically compute the force and torques that
+are produced at this point to ensure the equality constraint is met.
+
+For more information about constraints, see
+https://mujoco.readthedocs.io/en/stable
+
+Astute readers
```

Comment:
?? Did you mean to add more text, or are you just checking if I actually read this ;-)